src/domain/utils/PPSProgramsHelper.ts

```diff
@@ -65,7 +65,7 @@ export const getSurveyOptions = (
                     return ["Edit", "Add new Hospital", "List Hospitals"];
                 case "SUPRANATIONAL":
                 default:
-                    return ["Edit", "Add new Country", "List Countries"];
+                    return ["Edit", "Add new Country", "List Countries", "Delete"];
```

Comment:
@mariaozamiz  I just did some functional testing, we need to be able to delete all types of surveys not only SUPRANATIONAL + PPSSurveyForm surveys. Please add delete option to all survey types and test is as well.